test/wallet-accounts-reveal-test.js

```diff
@@ -0,0 +1,163 @@
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+
+'use strict';
+
+const assert = require('bsert');
+const Network = require('../lib/protocol/network');
+const FullNode = require('../lib/node/fullnode');
+const Address = require('../lib/primitives/address');
+const rules = require('../lib/covenants/rules');
+const {WalletClient} = require('hs-client');
+
+const network = Network.get('regtest');
+
+const node = new FullNode({
+  memory: true,
+  network: 'regtest',
+  plugins: [require('../lib/wallet/plugin')]
+});
+
+const wclient = new WalletClient({
+  port: network.walletPort
+});
+
+const {wdb} = node.require('walletdb');
+const name = rules.grindName(10, 20, network);
+
+let wallet, alice, bob, aliceMiner, bobMiner;
+
+async function mineBlocks(n, addr) {
```

Comment:
#168 introduces a `common` util for mining blocks: https://github.com/handshake-org/hsd/pull/168/files#diff-e169f8c8ca803ee76312e094ba6f17cbR120

I don't know what the convention is for setting up mining in tests - but thought it was worth a mention :bowing_man: man:

Reply:
This is cool too but again, requires the client which I'm trying to avoid for this patch. The function in my test also has a quick-n-dirty default value for the miner address when I'm not trying to fund a specific wallet account.